### DISCUSSION TITLE
Is farm support bun?

### DISCUSSION THREAD

**Discussion by muazhari** [farm-fe/farm#1193] [Q&A]
When I'm using `process.env` or `bun.env`, not defined error came up. I tried the work around by using `bunx --bun farm start` or `bunx farm start`. For the former, it leads to not starting the commad. For the latter, it still has not defined error.

**Comment by wre232114** [comment]
Is there a repository or steps that can reproduce the error? Farm is not tested against Bun, we need more info to locate the error

**Reply by muazhari** [reply]
To reproduce, just use the `process.env` or `bun.env` in the react code created by `bun create farm`, then run it.

**Reply by wre232114** [reply]
Thanks, I'll try

**Reply by muazhari** [reply]
There is an update. The error is already fixed. The error coming up because I am using the undefined env variable. Somehow, the error only informing the process/bun is not being defined and not explicitly inform because of an undefined env variable.
![image](https://github.com/farm-fe/farm/assets/39398937/0825b1f0-1cc2-448d-bec0-f0f1041fd3d9)

**Reply by wre232114** [reply]
We'll find a better way to display the error later
